extension/oauth2clientcredentialsauthextension/README.md

```diff
@@ -0,0 +1,53 @@
+# Authenticator - OAuth2 Client Credentials
+
+This extension implements a `configauth.ClientAuthenticator` and is to be used by HTTP and gRPC based exporters.
+The authenticator type has to be set to `oauth2`.
+
+## Configuration
+
+```yaml
+extensions:
+  oauth2clientcredentials:
```

Comment:
Can we shorten the name a bit? How about `oauth2client` or even simply `oauth2`? Do we expect that there will be `oauth2` ServerAuth extension that will be implemented separately from this one? I think we are able to implement both ServerAuth and ClientAuth in one extension, right? If I am not wrong there are no conflicting methods that prevent this.

Reply:
Thanks @tigrannajaryan - I have renamed the package to oauth2client.  Oauth2 has 2 other workflows - And from our collector perspective we are always acting as  a "client"  directly as opposed to acting on behalf of a user in other 2 use cases. Kinldy let me know.